Issue 28 38 39

chaneylc opened the pull request:
issues #28 #38 #39

chaneylc committed:
Added setting to bypass coin measurement with manual input.
Added new measurement options algorithms and settings.
Refactored image processing functions.
Added visualizations for min and max axis.
Fixed bottom nav icon not switching after importing a file to process.

chaneylc committed:
Auto switch to slow measurement when LSS is selected.

chaneylc committed:
Merge remote-tracking branch 'origin/dev' into issue_28_38_39

# Conflicts:
#	onekk/src/main/java/org/wheatgenetics/imageprocess/DetectWithReference.kt
#	onekk/src/main/res/values/strings.xml

trife commented:
- Create `Analysis` settings section above workflow and move the `Algorithm` and `Measurement` settings to it
- Merge `Reference` and `Coin` sections (behavior here is as expected)
- If user selects LSS algorithm, prompt them to select the slow method with the following: "Large, non-symmetric objects are poorly measured with the Fast/Medium measuring settings. Would you like to switch to the slow method?"

chaneylc committed:
Create Analysis settings section above workflow and move the Algorithm and Measurement settings to it
Merge Reference and Coin sections (behavior here is as expected)
If user selects LSS algorithm, prompt them to select the slow method with the following: "Large, non-symmetric objects are poorly measured with the Fast/Medium measuring settings. Would you like to switch to the slow method?"

trife committed:
Updated changelog
